packages/cli/src/models/network/TransactionController.ts

```diff
@@ -55,18 +56,27 @@ export default class TransactionController {
     }
   }
 
-  public async sendTransaction(proxyAddress: string, methodName: string, methodArgs: string[]): Promise<any> {
-    if (!this.networkFile.hasProxies({ address: proxyAddress })) {
-      throw Error(`Proxy at address ${proxyAddress} not found.`);
-    }
+  public async callContractMethod(proxyAddress: string, methodName: string, methodArgs: string[]): Promise<void | never> {
+    const { method, contract } = this.getContractAndMethod(proxyAddress, methodName, methodArgs);
+    try {
+      log.info(`Calling: ${callDescription(method, methodArgs)}`);
+      let result = await contract.methods[methodName](...methodArgs).call({ ...this.txParams });
+      if (typeof result === 'object' && Object.keys(result).length !== 0) {
+        result = `(${Object.values(result).join(', ')})`;
+      }
 
-    const { package: packageName, contract: contractName } = this.networkFile.getProxy(proxyAddress);
-    const contractManager = new ContractManager(this.packageFile);
-    const contract = contractManager.getContractClass(packageName, contractName).at(proxyAddress);
-    const { method } = buildCallData(contract, methodName, methodArgs);
+      isUndefined(result) || typeof result === 'object'
+        ? log.info(`Method ${methodName} successfully called.`)
+        : log.info(`Call returned: ${result}`);
```

Comment:
What do you think about, instead of logging the result here, return the value from the `controller` to the `script`? And have the script output the result to `stdout`, so it can be piped if needed? It would be similar to how `create` works in outputting the deployment address.

Reply:
We can do both!